benchmarks/web-search/server/Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM cloudsuite/java
-MAINTAINER Javier Picorel <javier.picorel@epfl.ch>
+MAINTAINER Sidharth Gupta <sidharth.gupta@epfl.ch>
```

Comment:
You forgot to fix this one.
Same for the client image of web-search.

Reply:
done, good catch, thanks!

Reply:
You forgot the client. 🤦‍♂️

Reply:
sorry, now it's fine